### DISCUSSION TITLE
New releases of semantic-release throwing typescript error

### DISCUSSION THREAD

**Discussion by blackfalcon** [AlaskaAirlines/WC-Generator#553] [Announcements]
### General Support Request

See the following requested dependency updates

```
 Minor Update New backwards-compatible features.
❯◯ semantic-release devDep  23.0.2  ❯  23.1.1  https://github.com/semantic-release/semantic-release#readme

 Major Update Potentially breaking API changes. Use caution.
 ◯ @semantic-release/npm devDep     11.0.3  ❯  12.0.1  https://github.com/semantic-release/npm#readme
```

When updating to these new versions, you will see the following error

```
> @aurodesignsystem/auro-datepicker@2.5.2 types
> tsc

node_modules/mime/dist/src/Mime.d.ts:5:5 - error TS18028: Private identifiers are only available when targeting ECMAScript 2015 and higher.

5     #private;
      ~~~~~~~~


Found 1 error in node_modules/mime/dist/src/Mime.d.ts:5
```

### Possible Solution

Possibly wait until next release of semantic-release of try to understand this obscure typescript error. 

### Additional context

Typescript is hipster scam! 

### Exit criteria

This issue can be closed once the build error has been mitigated.

**Comment by jason-capsule42** [comment]
This can be resolved by adding the following to config options to our `tsconfig` file.

```json
    "target": "ES2022",
    "moduleResolution": "node"
```
